src/schema/provenance_schema.yaml

```diff
@@ -192,6 +192,10 @@ shared_entity_properties: &shared_entity_properties
 ENTITIES:
   ############################################# Collection #############################################
   Collection:
+    excluded_properties_from_public_response:
+      - datasets:
+        - lab_dataset_id
+        - submission_id
```

Comment:
- Remove `submission_id` since dataset doesn't have such field
- Add `metadata.lab_id`, same as https://github.com/hubmapconsortium/entity-api/blob/c367217e7d6355091692072bf4d8cabee75f80b4/src/schema/provenance_schema.yaml#L309-L310

Reply:
For my Collection test data, the Datasets inside `datasets` did not have `metadata`. But I have made the change.